src/mame/drivers/pgm2.cpp

```diff
@@ -34,7 +34,7 @@
     Dodonpachi Daioujou Tamashii
     Knights of Valour 3
 
-    These were only released as single board PGM2 based hardware, seen for sale in Japan for around $250-$300
+    These were only released in Japan, seen for sale for around $250-$300
```

Comment:
What about the errors almost immediately below this where it calls Ocha-Ken no Puzzle all kinds of wrong things?

Reply:
Totally missed that. Pushed a fix for that too!